src/main/java/com/devcourse/checkmoi/global/annotation/ValueOfEnumValidator.java

```diff
@@ -0,0 +1,35 @@
+package com.devcourse.checkmoi.global.annotation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class ValueOfEnumValidator implements ConstraintValidator<ValueOfEnum, String> {
+
+    private ValueOfEnum annotation;
+
+    @Override
+    public void initialize(ValueOfEnum constraintAnnotation) {
+        this.annotation = constraintAnnotation;
+    }
+
+    @Override
+    public boolean isValid(String value, ConstraintValidatorContext context) {
+        if (value == null) {
+            return true;
+        }
+        boolean result = false;
+        Object[] enumValues = this.annotation.enumClass().getEnumConstants();
+        if (enumValues != null) {
```

Comment:
다음과 같이 하고 밑에 있는 부분을 한 뎁스 줄이는 것은 어떠실까요?
```suggestion
    if (enumValues == null) {
        return false;
    }
```